test/lib/creation.go

```diff
@@ -566,9 +567,9 @@ func (c *Client) CreateClusterRoleBindingOrFail(saName, crName, crbName string)
 
 func (c *Client) applyAdditionalEnv(pod *corev1.PodSpec) {
 	for i := 0; i < len(pod.Containers); i++ {
-		pod.Containers[i].Env = append(pod.Containers[i].Env, c.tracingEnv)
-		if c.loggingEnv != nil {
-			pod.Containers[i].Env = append(pod.Containers[i].Env, *c.loggingEnv)
+		pod.Containers[i].Env = append(pod.Containers[i].Env, corev1.EnvVar{Name: ti.ConfigTracingEnv, Value: c.TracingCfg})
+		if c.loggingCfg != "" {
+			pod.Containers[i].Env = append(pod.Containers[i].Env, corev1.EnvVar{Name: ti.ConfigLoggingEnv, Value: c.loggingCfg})
 		}
```

Comment:
Perhaps we can refactor logging and tracing in \[1\] to be consistent when `""` is provided because it looks weird that we're handling the empty string only for the logging config.

\[1\]:
https://github.com/knative/eventing/blob/eb2bfff3eb9802ad1509756c1de7053a8e8bde30/test/test_images/utils.go#L59-L85

Reply:
@pierDipi Did you mean something like this? https://github.com/knative/eventing/pull/6289